.github/workflows/iotlab-ci.yml

```
@@ -0,0 +1,116 @@
+name: iotlab-ci
+
+# Run 'compile_and_test_for_board.py' on IoT-LAB boards.
+#
+# This workflow will launch all tests on a list of IoT-LAB boards.
+#
+# This workflow is only run periodically on master with a cron and on new
+# pushed release candidates tags.
+
+on:
+  # Schedule weekly runs on Monday at 00:00 on master
+  schedule:
+    - cron: '00 0 * * 1'
+  push:
+    # Run on all new release candidates tags
+    tags:
+      - '[0-9][0-9][0-9][0-9]-RC[0-9]*'
```

Oops, there is now the month missing :sweat_smile: 

```suggestion
      - '[0-9][0-9][0-9][0-9].[0-9][0-9]-RC[0-9]*'
```

---

Also does it make sense to also build for releases and point releases?

```suggestion
      - '[0-9][0-9][0-9][0-9].[0-9][0-9]-RC[0-9]*'
      - '[0-9][0-9][0-9][0-9].[0-9][0-9]'
      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*'
```

---

Fixed